### PR TITLE
calling Alloy from Haskell

### PR DESCRIPTION
**Pull request by marcellussiegburg**
Make it so that in `random-cd2alloy` there is a call to the Alloy Analyzer at the end (in a suitable form, maybe a Java executable wrapping Alloy), on the concatenation of files `output.part1` ... `output.part5`, and all (or a certain number of) instances in text form are generated.

### CODE REVIEW

**Comment by jvoigtlaender** [comment]
This is now to happen in `random-stuff` instead of `random-cd2alloy`.

**Comment by jvoigtlaender** [comment]
And the relevant file is directly the complete "output.als", not in parts anymore.

**Comment by jvoigtlaender** [comment]
Do it by changing the `giveMeInstances` stub I just added and committed.

File: random-stuff.hs [hunk 1]
```diff
@@ -1,38 +1,54 @@
+{-# LANGUAGE NamedFieldPuns #-}
+
```
Comment (jvoigtlaender) [inline random-stuff.hs]: what for?

File: random-stuff.hs [hunk 2]
```diff
@@ -1,38 +1,54 @@
...
+  drawCdFromSyntax syntax (output config) Pdf
+  unless (anyRedEdge syntax) $ do
     time <- getZonedTime
     let (part1, part2, part3, part4, part5) = transform syntax "" (show time)
-    let out = output ++ ".als"
+    let out = output config ++ ".als"
     writeFile out (part1 ++ part2 ++ part3 ++ part4 ++ part5)
-    putStrLn ("More output written to " ++ out)
-    instances <- giveMeInstances
+    putStrLn ("More output written to " ++ output config)
```
Comment (jvoigtlaender) [inline random-stuff.hs]: Isn’t out the correct file name?

File: random-stuff.hs (hunk 1 above)
Comment (marcellussiegburg) [reply random-stuff.hs]: I guess I missed that on manual merging without history. It is not required.

**Review by jvoigtlaender — COMMENTED** [review]
Thanks

**Pull request merged**